### PR TITLE
Initial fuel emissions factors

### LINKED ISSUE #33: Make CO2 calculations based on liters
The API calculates CO2 emission based on mileage per mode of transport and people travelling, which is super useful! However, some platforms (e.g. tank cards) don't have mileage/distance/location information, only how many liters of a certain fuel are tanked. 

Maybe this API could also calculate CO2 emission based on the amount and type of fuel tanked. Data on how much CO2 is emitted per fuel type can be found on https://www.co2emissiefactoren.nl/lijst-emissiefactoren/

p.s.
I don't have any knowledge on coding, so not sure if this is possible to implement, just someone who is passionate about MaaS. 

# Addendum
*This section was added after the initial issue was created, in order to add information and resources related to the task.*

## Resources
- [Ecoscore](https://ecoscore.be/en/info/ecoscore/co2) - describes a method of calculating CO2 emissions per litre of petrol and diesel
- [table of CO2 emissions by fuel](https://people.exeter.ac.uk/TWDavies/energy_conversion/Calculation%20of%20CO2%20emissions%20from%20fuels.htm) without mention of source or methodology (may still prove useful)
- Natural Resources Canada published a [fact sheet estimating CO2 kg/L for several fuels](https://www.nrcan.gc.ca/sites/www.nrcan.gc.ca/files/oee/pdf/transportation/fuel-efficient-technologies/autosmart_factsheet_6_e.pdf) - namely petrol, diesel, biodiesel, and ethanol
- [IPCC emissions factors database](https://www.ipcc-nggip.iges.or.jp/EFDB/main.php) has extensive data on [road transportation fuel combustion activities](https://www.ipcc-nggip.iges.or.jp/EFDB/find_ef.php?ipcc_code=1.A.3.b&ipcc_level=3)
- U.S. Environmental Protection Agency published [CO2 emissions metrics for gasoline and diesel combustion](https://www.epa.gov/energy/greenhouse-gases-equivalencies-calculator-calculations-and-references) as well as a [greenhouse gas equivalencies calculator](https://www.epa.gov/energy/greenhouse-gas-equivalencies-calculator)

### PR DESCRIPTION
**Pull request by brylie**
Closes #33 

Follow-up PR for adding API: #44

### CODE REVIEW

**Comment by MarkusShepherd** [comment]
Very good start! 😎 I'm still wondering how we'll incorporate fuels that aren't measured in volume (as per the discussion in #33), but for a start this seems very good (particularly in the spirit of a timely delivery).

Looks like Python doesn't do well with one argument constructors (at least I could get the original code to work), so I've pushed an update that should work.

If we add the method API and some tests for this new class, we should be good to go for the next iteration.

**Comment by brylie** [comment]
@MarkusShepherd do you think we might want to deprecate the method API in a separate task? It seems a bit redundant.

**Comment by brylie** [comment]
@MarkusShepherd I added the method API and unit tests. Please review.

File: estimator/transport_co2/tests/test_transport_co2_estimator.py [hunk 1]
```diff
@@ -62,3 +65,52 @@ def test_estimate_co2_scooter_with_occupancy():
...
+
+def test_estimate_fuel_co2_g_biodiesel_20():
+    co2_estimate = estimate_fuel_co2_g(fuel="biodiesel_20", litres=1)
+
+    assert co2_estimate == approx(2_620)
+
+# custom co2_g_per_litre
+
+
+def test_estimate_fuel_co2_g_biodiesel_20():
```
Comment (MarkusShepherd) [inline estimator/transport_co2/tests/test_transport_co2_estimator.py:111]: That's the same method name as before.

File: estimator/transport_co2/estimator.py [hunk 2]
```diff
@@ -32,3 +32,27 @@ def estimate_co2(
         co2_per_vehicle_km=co2_per_vehicle_km,
         occupancy=occupancy,
     )
+
+
+def estimate_fuel_co2_g(
+    fuel: Union[str, Mode],
```
Comment (MarkusShepherd) [inline estimator/transport_co2/estimator.py:38]: This should be `Fuel`.

**Pull request merged**